diffJobSettings.py

```diff
@@ -1,51 +1,105 @@
-#!/usr/bin/python
+#!/usr/bin/python3
 
 import subprocess
 import argparse
 import sys
 import requests
 import json
 
-parser = argparse.ArgumentParser()
-parser.add_argument('--host')
-parser.add_argument('--jobids')
-parser.add_argument('--nossl', action='store_true')
+parser = argparse.ArgumentParser(description='openqa test data:display the  differences of Settings betwen 2 given tests IDs Or display the full data set of a single test ID')
+parser.add_argument('--host', default='openqa.suse.de',help='default openqa.suse.de')
+parser.add_argument('--jobids','-j', help='jobid1[,jobid2] # Enter 1[Full data] or 2[Differences] IDs')
+parser.add_argument('--nossl','-n', action='store_true', help='http urls; default https')
+parser.add_argument('--comments','-c', action='store_true', help='print job comments too')
 args = parser.parse_args()
 
 source_url=''
-obvious_differs=("NAME")
```

Comment:
why you have dropped this variable ?

Reply:
As I understood, it was needed to exclude NAME from difference list (`continue`).
In this update I included the NAME parameter too in the list of data, to show all the differences, therefore this variable and related check-if was no more needed.

Reply:
But eventually I could restore and leave EMPTY that list:` obvious_differs=()`

Reply:
skip-list applied

Reply:
when I was using first versions of this script I have seen a lot of variables which *always* different and knowing that they different does not bring any valuable info .  For example there is "WORKER_ID" variable ofc. it will differ from run to run because certain worker picked up randomly from currently available and it does not help you anyhow knowing that yet another random value is selected .  Same goes to test "NAME" , it will be *always* different because NAME variable contains job ID so name is unique .
Main purpose of this script ( at least for me ) is to compare failed job with same one but passed,  to try to find single one which cause the failure . The purpose of "obvious_differes" variables is to remove the "noise" from the list . To see only variables which really have chance to be failure cause and NAME have almost no chance to be that variable ( as many others , I planned to extend this list but never had a chance ... )

Reply:
I agree with your POV. I restored the code, but let empty the list: In this case I gonna restore the original settings (NAME) and eventually each one local copy can be changed on own need. 
Future implementations could eventually provide a more option with fileds to skip :-)

Reply:
I don't see this changes yet , maybe your forgot to push them ?

Reply:
changes ongoing ... :) will be online soon

Reply:
Please, see the last commit: added skip default 'NAME' and a new option to eventually enter more fileds  to skip.

Reply:
Consider we that the job json has up to 3 levels (`job.level1.level2`) dictionary and actual diff is only performed in the` job.settings.`. Therefore the current excluded fields can be in that 3rd level only, none in other 2nd levels as job.assets or job.state, in this implementation :)